internal/provider/data_source_vault_cluster.go

```diff
@@ -192,6 +212,26 @@ If a project is not configured in the HCP Provider config block, the oldest proj
 							Type:        schema.TypeString,
 							Computed:    true,
 						},
+						"cloudwatch_access_key_id": {
+							Description: "CloudWatch access key ID for streaming audit logs",
+							Type:        schema.TypeString,
+							Computed:    true,
+						},
```

Comment:
Looks like we need `cloudwatch_secret_access_key` here as well?

Reply:
done, funny how the code for preventing the value from the state file from being updated with "redacted" preventing this issue from surfacing in my tests :laughing: